libs/wingsdk/src/shared/misc.ts

```diff
@@ -24,16 +27,14 @@ export function normalPath(path: string) {
  * Just a helpful wrapper around `execFile` that returns a promise.
  */
 export async function runCommand(cmd: string, args: string[]): Promise<any> {
-  const raw = await new Promise((resolve, reject) => {
-    execFile(cmd, args, (error, stdout, stderr) => {
-      if (error) {
-        console.error(stderr);
```

Comment:
Since we're no longer calling console.error, we should make sure we aren't breaking any existing usages of runComand. i.e. if they were expecting this to write the error we'll need to ensure the error still get's written somewhere